### PR TITLE
[Zhou Xinming] Duke Increments

### PR DESCRIPTION
**Pull request by xinmingzh**

### CODE REVIEW

File: src/main/java/duke/task/Event.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Event extends Task {
+    private static final Pattern PAT = Pattern.compile("(.+) /at (.+)  (.+)");
+    private static final DateTimeFormatter readFormatter = DateTimeFormatter.ofPattern("d/M/yyyy HHmm", Locale.ENGLISH);
```
Comment (chanjunren) [inline src/main/java/duke/task/Event.java]: Hi bro,
the names of final variables should be all caps :D

File: src/main/java/duke/task/Event.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+     * @param endDate Event end datetime
+     * @param done Event done state
+     */
+    public Event(String desc, LocalDateTime startDate, LocalDateTime endDate, boolean done) {
+        super(desc, done);
+        this.startDate = startDate;
+        this.endDate = endDate;
+    }
+
+    public static Task parse(String commandContent) throws DukeException {
```
Comment (KxxMxxx) [inline src/main/java/duke/task/Event.java]: A Javadoc comment should be added here to describe what the method does.

**Review by KxxMxxx — COMMENTED** [review]
I think your code logic is good! Your code is very clean too!

**Review by chanjunren — COMMENTED** [review]
(Sorry I forgot to press submit review previously)

**Pull request closed without merging**